CIScripts/Test/TestRunner.ps1

```diff
@@ -45,7 +45,9 @@ function Run-TestScenarios {
         # Test-DockerDriverMultiTenancy -Session $Sessions[0] -TestConfiguration $TestConfiguration
         Test-WindowsLinuxIntegration -Session $Sessions[0] -TestConfiguration (Get-TestConfigurationWindowsLinux)
         # Test-Pkt0PipeImplementation -Session $Sessions[0] -TestConfiguration $TestConfiguration
-        Invoke-Pester -Script @{Path=".\Test\Tests\Pkt0.Tests.ps1"; Parameters=@{TestbedAddr=$Sessions[0].ComputerName}}
+        if($Env:RUN_PESTER_TESTS -eq "1") {
+            Invoke-Pester -Script @{Path=".\Test\Tests\Pkt0.Tests.ps1"; Parameters=@{TestbedAddr=$Sessions[0].ComputerName}}
```

Comment:
When some tests fail, how this failure is propagated up?

Also, make sure if `Invoke-Pester` doesn't set `$LastExitCode`. If it does, it has to be wrapped in `Invoke-NativeCommand` (why? tldr: Jenkins pipeline powershell wrapper is weird)